lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntime.cpp

```diff
@@ -576,8 +574,15 @@ bool SwiftLanguageRuntimeImpl::AddJitObjectFileToReflectionContext(
       log->Printf("%s: Could not find out swift reflection section names for "
                   "object format type.",
                   __FUNCTION__);
-    return false;
   }
+  return obj_file_format;
+}
+
+bool SwiftLanguageRuntimeImpl::AddJitObjectFileToReflectionContext(
+    ObjectFile &obj_file, llvm::Triple::ObjectFormatType obj_format_type) {
+  assert(obj_file.GetType() == ObjectFile::eTypeJIT &&
+         "Not a JIT object file!");
+  auto obj_file_format = GetObjectFileFormat(obj_format_type);
```

Comment:
it looks like this can return a null unique_ptr, but in the closure below we just access it with -> without checking?